test/jdk/java/awt/MenuItem/LotsOfMenuItemsTest.java

```diff
@@ -42,20 +40,20 @@
  * @run main/manual LotsOfMenuItemsTest
  */
 
-public class LotsOfMenuItemsTest implements ComponentListener {
+public class LotsOfMenuItemsTest extends ComponentAdapter {
     private static final int NUM_WINDOWS = 400;
-    private static TestFrame firstFrame, testFrame;
-    private static Rectangle rect;
+    private static TestFrame firstFrame;
+    private static TestFrame testFrame;
 
     public static void main(String[] args) throws Exception {
         LotsOfMenuItemsTest obj = new LotsOfMenuItemsTest();
         String INSTRUCTIONS = """
-                This test creates a lots of frames with menubars.
+                This test creates lots of frames with menubars.
```

Comment:
```suggestion
                This test creates lots of frames with menu bars.
```
It's usually spelt with a space.

Reply:
Ok.. updated.